src/storage/storage_structure/disk_array.cpp

```diff
@@ -165,6 +165,7 @@ uint64_t BaseDiskArrayInternal::pushBack(std::span<uint8_t> val) {
 }
 
 uint64_t BaseDiskArrayInternal::resize(uint64_t newNumElements, std::span<uint8_t> defaultVal) {
```

Comment:
Do we also need to lock in `BaseDiskArrayInternal::pushBack(std::span<uint8_t> val)`? (L160)

Reply:
Yes, thanks for catching that.